rhaptos/cnxmlutils/xsl/test/title.cnxml.html

```diff
@@ -190,6 +190,59 @@
       </p>
     </div>
   </section>
+<!-- m44388 note with 1 para -->
+  <section
+    data-depth='1'
+    id='noted-section-1'
+  >
+    <div
+      class='note career'
+      data-has-label='true'
+      data-label=''
+      data-type='note'
+      id='fs-id2681692'
+    >
+      <div
+        class='title'
+        data-type='title'
+      >Career Connection</div>
+      <p
+        id='fs-id1808500'
+      >
+        <span
```

Comment:
Any thoughts on this format (having a blockish title inside a `p` which normally contains only inline text). This might also affect the WYSIWYG editor (maybe `<para>` with `<title>` should be transformed to a: 

```
<div data-type='para'>
  <span data-type='title'/>
  <p/>
</div>
```

Thoughts on impact?